Automattic/wp-calypso
# Janitorial: Refactor Register Domain Step (Signup/Search)

- Migrate all domains components to `prop-types`

Register Domain Step
- Remove mixin
- move to redux actions
- remove code for tracking time between searches

Signup Domains Step:
- localize and remove mixin

Test:
- Go to `Domains -> Add`
- Search for a domain
- Make sure results are present
- Click on a domain
- Make sure it's tracked
- Click on mapping
- Make sure it's tracked

Test Signup:
- Go calypso.localhost:3000/start/
- Get to Domains step
- Search for something
- Make sure results are present
- Click on something
- Make sure it's tracked
